### PR TITLE
Enhancement: Refactor the existing direct load implements. 

### LINKED ISSUE #86: [Enhancement]: Introduce the latest version of direct load API and refactor the existing direct load implements.
### Description

[obkv-table-client](https://github.com/oceanbase/obkv-table-client-java/releases/tag/obkv-table-client-1.2.13) has released a new version with many improvements.
However, the direct load API has some breaking changes, so we need to refactor the existing direct load implements to adapt to the new API.

Once this issue is fixed, we can start working on #61.

### PR DESCRIPTION
**Pull request by yuanoOo**
## Summary

Closes #86 

## Solution Description

1. Upgrade obkv-table-client to the latest version.
2. Refactor the existing direct load implements.

### CODE REVIEW

File: docs/sink/flink-connector-oceanbase.md [hunk 1]
```diff
@@ -148,31 +148,33 @@ CREATE TABLE t_sink
...
+| direct-load.load-method        | No                    | No                     | full                     | String   | Direct load load mode: full, inc, inc_replace. full: full direct load, default value. inc: normal incremental direct load, primary key conflict check will be performed, observer-4.3.2 and above support, dupAction REPLACE is not supported for the time being.inc_replace: special replace mode incremental direct load, no primary key conflict check will be performed, directly overwrite the old data (equivalent to the effect of replace), dupAction parameter will be ignored, observer-4.3.2 and above support. |
```
Comment (whhe) [inline docs/sink/flink-connector-oceanbase.md:177]: You can use some html tags like `<li>` and `<code>` to rearrange the description.

**Review by whhe — APPROVED** [review]
Overall LGTM. For the table in docs, I think it's OK to migrate it from markdown to html, otherwise we may need to update the whole section for many times in the future. WDYT? @yuanoOo

**Comment by yuanoOo** [comment]
> Overall LGTM. For the table in docs, I think it's OK to migrate it from markdown to html, otherwise we may need to update the whole section for many times in the future. WDYT? 总体而言LGTM。对于docs中的表格，我认为将其从markdown迁移到html是可以的，否则我们将来可能需要多次更新整个部分。 WDYT？@yuanoOo

Agreed, I will convert it to html format.

File: docs/sink/flink-connector-oceanbase.md (hunk 1 above)
Comment (yuanoOo) [reply docs/sink/flink-connector-oceanbase.md]: Converted to html format.

**Review by whhe — APPROVED** [review]
LGTM

**Pull request merged**